source/postprocess/dynamic_topography.cc

```diff
@@ -294,105 +484,23 @@ namespace aspect
-
-              file << "# "
-                   << ((dim==2)? "x y" : "x y z")
-                   << " lower surface topography" << std::endl;
-
-              // first write out the data we have created locally
-              file << output_lower.str();
-
-              std::string tmp;
-              tmp.resize (max_data_length, '\0');
-
-              // then loop through all of the other processors and collect
-              // data, then write it to the file
-              for (unsigned int p=1; p<Utilities::MPI::n_mpi_processes(this->get_mpi_communicator()); ++p)
-                {
-                  MPI_Status status;
-                  // get the data. note that MPI says that an MPI_Recv may receive
-                  // less data than the length specified here. since we have already
-                  // determined the maximal message length, we use this feature here
-                  // rather than trying to find out the exact message length with
-                  // a call to MPI_Probe.
-                  MPI_Recv (&tmp[0], max_data_length, MPI_CHAR, p, mpi_tag,
-                            this->get_mpi_communicator(), &status);
-
-                  // output the string. note that 'tmp' has length max_data_length,
-                  // but we only wrote a certain piece of it in the MPI_Recv, ended
-                  // by a \0 character. write only this part by outputting it as a
-                  // C string object, rather than as a std::string
-                  file << tmp.c_str();
-                }
-            }
-          else
-            // on other processors, send the data to processor zero. include the \0
-            // character at the end of the string
-            {
-              MPI_Send (&output_lower.str()[0], output_lower.str().size()+1, MPI_CHAR, 0, mpi_tag,
-                        this->get_mpi_communicator());
-            }
-        }
-
-      return std::pair<std::string,std::string>("Writing dynamic topography:",
-                                                filename);
     }
 
+
+    /**
+     * Declare the parameters for the postprocessor.
+     */
     template <int dim>
     void
-    DynamicTopography<dim>::
-    declare_parameters (ParameterHandler &prm)
+    DynamicTopography<dim>::declare_parameters (ParameterHandler &prm)
     {
       prm.enter_subsection("Postprocess");
       {
         prm.enter_subsection("Dynamic Topography");
         {
-          prm.declare_entry ("Subtract mean of dynamic topography", "false",
-                             Patterns::Bool (),
-                             "Option to remove the mean dynamic topography "
-                             "in the visualization. The mean dynamic topography is calculated "
-                             "and subtracted separately for the upper and lower boundary. "
-                             "'true' subtracts the mean, 'false' leaves "
-                             "the calculated dynamic topography as is.");
```

Comment:
The intention is that it is no longer necessary with a more accurate dynamic topography solution. The `Subtract mean of dynamic topography` option was a bit of a workaround for poor accuracy in the pressure and stress.